sbysrc/sby_core.py

```diff
@@ -242,10 +242,11 @@ def taskloop(self):
                 if task.running:
                     fds.append(task.p.stdout)
 
-            try:
-                select(fds, [], [], 1.0) == ([], [], [])
-            except InterruptedError:
-                pass
+            if os.name == "posix":
+                try:
+                    select(fds, [], [], 1.0) == ([], [], [])
+                except InterruptedError:
+                    pass
```

Comment:
Don't we need an "else" branch here to avoid busy waiting?

Also, the `fds` array created above is only used in this select call. So if they aren't used anywhere else then probably generating this array should also be in the `if os.name == "posix"` block.

Reply:
on Windows select can work only on sockets, and not on files like in in this case. Still looking for proper implementation in case of Windows (sorry I did not mention that in pull request), but even without that part it actually makes sby works on Windows as is.

Reply:
ah, now I see that there was actually almost same PR for this https://github.com/YosysHQ/SymbiYosys/pull/56

Reply:
If there's no good way on windows to wait on a list of file descriptors then we should at least put something like `sleep(0.1)` in the else branch of this code.

Reply:
Will investigate more on this.  Would rather like proper implementation done of this.